dashed/forms.py

```diff
@@ -214,6 +214,7 @@ def __init__(self, viz):
                     '6 hour',
                     '1 day',
                     '7 days',
+                    # you could add a special string here
```

Comment:
say if you'd write "1 week, ending on sunday" here, you'd have to capture that on the other side on the `query` method and translate it into a granularity dict